### DISCUSSION TITLE
Stuck on opening assignment...

### DISCUSSION THREAD

**Discussion by EpicPhelan** [community/community#65480] [GitHub Education] [answered]
### Hi everyone,

It almost worked. I got the students in the Github Classroom and uploaded a template assignment then made the assignment. They installed the VScode extension for Github classroom and signed into their accounts in VScode. We can see the classroom and assignments. When they click the folder to open the assignment and edit it in VS code, a pop up in the bottom right says Opening Assignment, but never opens. It just hangs forever. Please help. My tech department blocked Replit, so I thought this was my next best option.

I see other people had this problem years ago, but all the threads say it was fixed.

**Comment by ssyberg** [comment]
Did you ever figure this out?

**Reply by EpicPhelan** [reply]
No. I gave up using Github classroom. I'm having to post files onto Google Class and having them upload the files back up after they are done. It hurts.

**Comment by joechea-aupp** [comment]
make sure your student have git installed on their computer. once it is installed, try to run git command from vscode terminal, if it ran, then access the assignment link again. if not reboot your computer and try git command on vscode again.

work for my class.

**Comment by juanpflores** [comment]
The extension has been deprecated since 2021. Students can still use the vscode for classroom assignment through the GitHub extension or the terminal.

**Comment by Zeeshan251** [comment]
Install GIT and try again. It'll work.

**Comment by tsollowa** [comment, marked as answer]
I installed Git and retried and it did not work initially, however, I restarted my PC and tried again and it worked! Thank you to those who commented.
